nengo_dl/converter.py

```diff
@@ -0,0 +1,1397 @@
+"""Tools for automatically converting a Keras model to a Nengo network."""
+
+import collections
+import logging
+import warnings
+
+import nengo
+import numpy as np
+import tensorflow as tf
+from tensorflow.python.keras.layers import BatchNormalization, BatchNormalizationV2
+from tensorflow.python.util import nest
+
+from nengo_dl.config import configure_settings
+from nengo_dl.tensor_node import Layer, TensorNode
+from nengo_dl.simulator import Simulator
+
+logger = logging.getLogger(__name__)
+
+
+class Converter:
+    """
+    Converts a Keras model to a Nengo network composed of native Nengo objects.
+
+    Parameters
+    ----------
+    model : ``tf.keras.Model``
+        Keras model to be converted
+    allow_fallback : bool
```

Comment:
I used to, but gave up on it because it wasn't used consistently (so just relying on the function signature to convey this information seemed more reliable).

Reply:
Makes sense!